app/screens/user_profile/user_info.tsx

```diff
@@ -17,36 +19,47 @@ type Props = {
     showNickname: boolean;
     showPosition: boolean;
     user: UserModel;
+    enableCustomAttributes?: boolean;
 }
 
-const UserInfo = ({localTime, showCustomStatus, showLocalTime, showNickname, showPosition, user}: Props) => {
-    const {formatMessage} = useIntl();
+const UserInfo = ({localTime, showCustomStatus, showLocalTime, showNickname, showPosition, user, enableCustomAttributes}: Props) => {
     const customStatus = getUserCustomStatus(user);
+    const serverUrl = useServerUrl();
+    const [customAttributes, setCustomAttributes] = useState<DisplayCustomAttribute[]>([]);
+
+    useEffect(() => {
+        if (enableCustomAttributes) {
+            const fetchData = async () => {
+                try {
+                    const client = NetworkManager.getClient(serverUrl);
+                    const fields = await client.getCustomProfileAttributeFields();
+                    const attrValues = await client.getCustomProfileAttributeValues(user.id);
+                    if (fields) {
+                        const attributes = fields.map((field) => ({
+                            id: field.id,
+                            name: field.name,
+                            value: attrValues[field.id] || '',
+                        }));
+                        setCustomAttributes(attributes);
+                    }
+                } catch {
+                    setCustomAttributes([]);
+                }
+            };
+
+            fetchData();
+        }
+    }, [enableCustomAttributes, serverUrl, user.id]);
 
     return (
         <>
-            {showCustomStatus && <UserProfileCustomStatus customStatus={customStatus!}/> }
-            {showNickname &&
-                <UserProfileLabel
-                    description={user.nickname}
-                    testID='user_profile.nickname'
-                    title={formatMessage({id: 'channel_info.nickname', defaultMessage: 'Nickname'})}
-                />
-            }
-            {showPosition &&
-                <UserProfileLabel
-                    description={user.position}
-                    testID='user_profile.position'
-                    title={formatMessage({id: 'channel_info.position', defaultMessage: 'Position'})}
-                />
-            }
-            {showLocalTime &&
-                <UserProfileLabel
-                    description={localTime!}
-                    testID='user_profile.local_time'
-                    title={formatMessage({id: 'channel_info.local_time', defaultMessage: 'Local Time'})}
-                />
-            }
+            {showCustomStatus && <UserProfileCustomStatus customStatus={customStatus!}/>}
+            <CustomAttributes
+                nickname={showNickname ? user.nickname : undefined}
+                position={showPosition ? user.position : undefined}
+                localTime={showLocalTime ? localTime : undefined}
+                customAttributes={enableCustomAttributes ? customAttributes : []}
```

Comment:
If we handle the change of `enableCustomAttributes` in the effect, we can simplify this (and not pass a new empty list on every render).

Also (not that is going to happen often...) if we keep this logic instead of resetting `customAttributes`, if the feature flag is switch off and back on, we may have for a few moments the old information, instead of the new one.